runtests.py

```diff
@@ -63,23 +63,71 @@
 
 assert all(cmd in cmds for cmd in FAST_FAIL)
 
-if not set(args).issubset(cmds):
-    print("usage:", prog, " ".join('[%s]' % k for k in cmds))
-    exit(1)
-
-if not args:
-    args = list(cmds)
-
-status = 0
 
-for arg in args:
-    cmd = cmds[arg]
-    print('run %s: %s' % (arg, cmd))
+def run_cmd(name: str) -> int:
+    status = 0
+    cmd = cmds[name]
+    print('run %s: %s' % (name, cmd))
     res = (system(cmd) & 0x7F00) >> 8
     if res:
-        print('\nFAILED: %s' % arg)
+        print('\nFAILED: %s' % name)
         status = res
-        if arg in FAST_FAIL:
+        if name in FAST_FAIL:
             exit(status)
+    return status
+
+
+def start_background_cmd(name: str) -> Popen:
+    cmd = cmds[name]
+    proc = subprocess.Popen(cmd,
+                            shell=True,
+                            stderr=subprocess.STDOUT,
```

Comment:
Could this cause mixing output of both processes? Maybe pipe both and then print when done.

Reply:
I think that this is already what happens. Both are piped to the same pipe.